modules/workbench/modules/reporting/assets/schemas/new_user_satisfaction_survey.json

```diff
@@ -0,0 +1,43 @@
+ [
+  {
+    "description": "Time snapshot was taken, in Epoch milliseconds. Same across all rows and all tables in the snapshot, and uniquely defines a particular snapshot.",
+    "name": "snapshot_timestamp",
+    "type": "INTEGER"
+  },
+  {
+    "name": "id",
+    "type": "INTEGER",
+    "description": "ID of the survey response.",
+    "mode": "NULLABLE"
+  },
+  {
+    "name": "user_id",
+    "type": "INTEGER",
+    "description": "ID of the user who created the survey response.",
+    "mode": "NULLABLE"
+  },
+  {
+    "name": "created",
+    "type": "TIMESTAMP",
+    "description": "Time the survey response was created.",
+    "mode": "NULLABLE"
+  },
+  {
+    "name": "modified",
+    "type": "TIMESTAMP",
+    "description": "Time the survey response was last modified.",
+    "mode": "NULLABLE"
+  },
+  {
+    "name": "satisfaction",
+    "type": "STRING",
```

Comment:
Have we discussed with the PDR team how they want to consume this data: as categories (like here) or as a numerical scale?  If it's the latter, we can make it easier on them by doing this conversion up front.  Or: we could provide this in both formats.

Reply:
The PDR team is ok with categories. They said if the reporting team requires a numerical scale they could make that conversion.

If this turns out to be false, providing the numeric format as an additional reporting-specific field is a great idea.